src/query/storages/fuse/src/statistics/column_statistic.rs

```diff
@@ -198,6 +201,56 @@ pub mod traverse {
         }
         Ok(())
     }
+
+    pub fn traverse_scalar_dns(
```

Comment:
is `dns` a type of search? or just depth first?

Reply:
> is `dns` a type of search? or just depth first?

depth first

Reply:
`depth first search` should named to `dfs`, not `dns`